bolt/outpost/settlement/v1/settlement.proto

```diff
@@ -265,25 +285,73 @@ message SwapExactInResponse {
 message WithdrawQuotesRequest {
   // base_asset: The base asset against which the accumulated quotes that the user wants to withdraw. e.g. USDT pool
   string base_asset = 1;
-}
-
-// WithdrawQuotesResponse: Response to the WithdrawQuotesRequest. Contains information about the withdrawn quotes.
+  // deposit_address: The address where withdrawn quotes should be deposit. If not provided, the default sender address
+  // would be used. Note it can be overwritten for specific quotes.
+  optional string deposit_address = 2;
+  // quotes: Details of how specific quotes should be withdrawn. Note that quotes that are not valid for this base, or
+  // were not collected would not cause an error - they will be simply ignored.
+  repeated QuoteWithdrawal quotes = 3;
+  // include_all_quotes: Determines if the quotes that are not included in `quotes` but were collected for this base
+  // should be also withdrawn using default withdrawal settings. It might be usefull to withdraw all the collected quotes,
+  // but specifying some details like deposit address only to limited subset of quotes. Setting this field with no `quotes`
+  // specified would result in error - when `quotes` are skipped, all the quotes are returned, and we need to make sure this
+  // field is not set to `false` accidently in which case the message would have no sense.
+  bool include_all_quotes = 4;
+}
+
+// WithdrawQuotesResponse: Response to the WithdrawQuotesRequest.
 message WithdrawQuotesResponse {
   // withdrawn: The amount of assets withdrawn. e.g. BTC, ETH
   repeated Asset withdrawn = 1;
-  // tx_id: The hash of the transaction.
-  string tx_id = 2;
 }
 
 // WithdrawAllQuotesRequest: Request to withdraw all quotes. Should be invoked by the liquidity provider to withdraw all quotes.
-message WithdrawAllQuotesRequest {}
+message WithdrawAllQuotesRequest {
+  // deposit_address: Address where withdrawn quotes should be deposit. If not provided, the default sender address would be used.
+  optional string deposit_address = 1;
+  // quotes: Details of how specific quotes should be withdrawn. Note, that regardless of this field, all the quotes would
+  // be withdrawn, not only specified - the quotes that are not specified here would use default values. Note that quotes that
+  // are not valid for this settlement, or were not collected would not cause an error - they will be simply ignored.
+  repeated QuoteWithdrawal quotes = 2;
```

Comment:
I think this request doesn't need any quotes configuration as its supposed to withdraw all the quotes, not just the ones passed in the request.
If a user wants to use the configuration and withdraw different quotes to different addresses, they can use the `WithdrawQuotes` instead. 
Else we will have two request which do almost similar things.

Reply:
This argument is meant to specify the receiver address per quote, and send other quotes to default receiver. But I see you don't like this idea in which case this field is redundant.